service/src/main/kotlin/fi/espoo/evaka/placement/PlacementType.kt

```diff
@@ -32,6 +35,20 @@ enum class PlacementType {
         SCHOOL_SHIFT_CARE -> false
     }
 
+    fun absenceCategories(): Set<AbsenceCategory> = when (this) {
+        CLUB -> setOf(AbsenceCategory.NONBILLABLE)
+        SCHOOL_SHIFT_CARE -> setOf(AbsenceCategory.NONBILLABLE)
+        PRESCHOOL, PREPARATORY -> setOf(AbsenceCategory.NONBILLABLE)
+        PRESCHOOL_DAYCARE, PREPARATORY_DAYCARE -> setOf(AbsenceCategory.BILLABLE, AbsenceCategory.NONBILLABLE)
+        DAYCARE -> setOf(AbsenceCategory.BILLABLE)
+        DAYCARE_PART_TIME -> setOf(AbsenceCategory.BILLABLE)
+        DAYCARE_FIVE_YEAR_OLDS, DAYCARE_PART_TIME_FIVE_YEAR_OLDS -> setOf(
+            AbsenceCategory.BILLABLE,
+            AbsenceCategory.NONBILLABLE
+        )
+        TEMPORARY_DAYCARE, TEMPORARY_DAYCARE_PART_DAY -> setOf(AbsenceCategory.BILLABLE)
+    }
```

Comment:
I think this is a bit hard to read since some of the placement types are grouped and some are not. Would it be clearer if they were divided into three categories? Something like this:
```
fun absenceCategories(): Set<AbsenceCategory> = when (this) {
    PRESCHOOL,
    PREPARATORY,
    CLUB,
    SCHOOL_SHIFT_CARE ->
        setOf(AbsenceCategory.NONBILLABLE)
    DAYCARE,
    DAYCARE_PART_TIME,
    TEMPORARY_DAYCARE,
    TEMPORARY_DAYCARE_PART_DAY ->
        setOf(AbsenceCategory.BILLABLE)
    PRESCHOOL_DAYCARE,
    PREPARATORY_DAYCARE,
    DAYCARE_FIVE_YEAR_OLDS,
    DAYCARE_PART_TIME_FIVE_YEAR_OLDS ->
        setOf(AbsenceCategory.BILLABLE, AbsenceCategory.NONBILLABLE)
}
```